tasks/volumes.yml

```diff
@@ -27,11 +27,26 @@
     {% endif %}
     -a {{ ansible_check_mode }}
   with_items: "{{ volumes }}"
-  when: item.type | default(libvirt_volume_default_type) == 'volume'
+  when: "{{ item.device | default(libvirt_volume_default_device) == 'disk' }}"
```

Comment:
This needs to use type rather than device. We support type=volume with device=cdrom.

Reply:
type in ['network', 'volume'] ?

Reply:
(see https://github.com/stackhpc/ansible-role-libvirt-vm/issues/45 for context)

Reply:
I've changed this line to use the `in` idiom, testing to follow.